src/main/java/com/thirdcc/webapp/service/EventAttendeeService.java

```diff
@@ -15,7 +15,7 @@
     /**
      * Save a eventAttendee.
      *
-     * @param eventAttendeeDTO the entity to save.
+     * @param eventAttendeeventeDTO the entity to save.
```

Comment:
Why rename like this?

Reply:
It's a typo. Issue solved in 2197a1fec0cf277016b55b2d815aea8a3c08ae05